library/nginx

```diff
@@ -3,72 +3,72 @@
 Maintainers: NGINX Docker Maintainers <docker-maint@nginx.com> (@nginxinc)
 GitRepo: https://github.com/nginxinc/docker-nginx.git
 
-Tags: 1.27.0, mainline, 1, 1.27, latest, 1.27.0-bookworm, mainline-bookworm, 1-bookworm, 1.27-bookworm, bookworm
+Tags: 1.27.1, mainline, 1, 1.27, latest, 1.27.1-bookworm, mainline-bookworm, 1-bookworm, 1.27-bookworm, bookworm
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-GitCommit: a6f7d140744f8b15ff4314b8718b3f022efc7f43
+GitCommit: e78cf70ce7b73a0c9ea734c9cf8aaaa283c1cc5a
 Directory: mainline/debian
 
-Tags: 1.27.0-perl, mainline-perl, 1-perl, 1.27-perl, perl, 1.27.0-bookworm-perl, mainline-bookworm-perl, 1-bookworm-perl, 1.27-bookworm-perl, bookworm-perl
+Tags: 1.27.1-perl, mainline-perl, 1-perl, 1.27-perl, perl, 1.27.1-bookworm-perl, mainline-bookworm-perl, 1-bookworm-perl, 1.27-bookworm-perl, bookworm-perl
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-GitCommit: 3180cdbec313dc4a9f6dd1109ae66adaf98f11fb
+GitCommit: e78cf70ce7b73a0c9ea734c9cf8aaaa283c1cc5a
 Directory: mainline/debian-perl
 
-Tags: 1.27.0-otel, mainline-otel, 1-otel, 1.27-otel, otel, 1.27.0-bookworm-otel, mainline-bookworm-otel, 1-bookworm-otel, 1.27-bookworm-otel, bookworm-otel
+Tags: 1.27.1-otel, mainline-otel, 1-otel, 1.27-otel, otel, 1.27.1-bookworm-otel, mainline-bookworm-otel, 1-bookworm-otel, 1.27-bookworm-otel, bookworm-otel
 Architectures: amd64, arm64v8
-GitCommit: 3180cdbec313dc4a9f6dd1109ae66adaf98f11fb
+GitCommit: e78cf70ce7b73a0c9ea734c9cf8aaaa283c1cc5a
 Directory: mainline/debian-otel
 
-Tags: 1.27.0-alpine, mainline-alpine, 1-alpine, 1.27-alpine, alpine, 1.27.0-alpine3.19, mainline-alpine3.19, 1-alpine3.19, 1.27-alpine3.19, alpine3.19
```

Comment:
This (and also stable) should be `alpine3.20` now; see also https://github.com/nginxinc/docker-nginx/pull/917

Reply:
Oh indeed, thanks for catching that!